### PR TITLE
Complete audio repository, auth fixes

### PR DESCRIPTION
**Pull request by MortonPL**
* GET `/health` - health check

* POST `/api/auth/login` - login w/ username and password
* GET `/api/auth/status` - check if logged
* POST `/api/auth/logout` - logout

* GET `/api/audio` - get all sample infos
* GET `/api/audio/{id}` - get sample data by id
* POST `/api/audio` - upload a sample in two field multipart
* DELETE `/api/audio/{id}` - remove sample by id

### CODE REVIEW

**Comment by MiniaczQ** [comment]
how about you GET some bitches

File: backend/src/main.rs [hunk 1]
```diff
@@ -25,6 +26,12 @@ async fn main() {
         surreal_db,
         file_storage,
     };
+
+    let repo = UserRepository {
+        surreal: SurrealDb::from_ref(&state.surreal_db),
+    }; // sucks
```
Comment (MiniaczQ) [inline backend/src/main.rs]: ok, dodaj sobie inicjalizator idk

File: backend/src/main.rs [hunk 2]
```diff
@@ -25,6 +26,12 @@ async fn main() {
...
+    let _ = repo.try_create("root", "root").await;
```
Comment (MiniaczQ) [inline backend/src/main.rs]:
```suggestion
    repo.try_create("root", "root").await.ok();
```

File: backend/src/routing/api/audio.rs [hunk 3]
```diff
@@ -102,13 +117,13 @@ async fn delete_audio(
 ///
 /// list all available audio sample identifiers
```
Comment (MiniaczQ) [inline backend/src/routing/api/audio.rs]: L

File: backend/src/routing/api/auth.rs [hunk 4]
```diff
@@ -76,12 +76,12 @@ async fn login(
...
     cookie.set_expires(None);
+    cookie.set_path("/api/");
     ResponseType::Data(cookies.add(cookie))
 }
 
-async fn logout() -> CookieJar {
-    let cookies = CookieJar::new();
-    cookies.remove(Cookie::from(JWT_TOKEN_COOKIE))
+async fn logout(jar: CookieJar) -> CookieJar {
+    jar.remove(Cookie::from(JWT_TOKEN_COOKIE))
```
Comment (MiniaczQ) [inline backend/src/routing/api/auth.rs:84]: jar(ed leto is a funny guy)

File: backend/src/services/database/repositories/user.rs [hunk 5]
```diff
@@ -13,7 +13,7 @@ use crate::services::database::{
 };
 
 pub struct UserRepository {
-    surreal: SurrealDb,
+    pub surreal: SurrealDb, // DEBUG
```
Comment (MiniaczQ) [inline backend/src/services/database/repositories/user.rs]: cool

**Review by MiniaczQ — CHANGES_REQUESTED** [review]

**Review by MiniaczQ — APPROVED** [review]
aprów

**Comment by MortonPL** [comment]
parów

**Pull request merged**